### PR TITLE
Change grafana dashboards for number of workspaces/users to be graphs with axes

### PR DESCRIPTION
**Pull request by tomgeorge**
### What does this PR do?

Changes the "Number of Users" and "Number of Workspaces" Grafana Panels to show axes, and add nice labels to the legend

### What issues does this PR fix or reference?
#14155 

Change grafana dashboards for number of workspaces/users to be graphs that show axes, legends

### Testing

```shell
oc apply -f deploy/openshift/templates/monitoring/grafana-dashboards.yaml -f deploy/openshift/templates/monitoring/grafana-datasources.yaml
oc process -f deploy/openshift/templates/monitoring/che-monitoring.yaml | oc apply -f -
# ensure CHE_METRICS_ENABLED=true
```

### CODE REVIEW

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 1]
```diff
@@ -3642,168 +3643,185 @@ data:
...
             "w": 5,
             "x": 0,
             "y": 1
           },
           "id": 14,
-          "interval": null,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
```
Comment (ibuziuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: could we have it set to true

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 2]
```diff
@@ -3642,168 +3643,185 @@ data:
...
+            "min": false,
```
Comment (ibuziuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: could we have it set to true

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 3]
```diff
@@ -3642,168 +3643,185 @@ data:
...
             "w": 5,
             "x": 5,
             "y": 1
           },
           "id": 47,
-          "interval": null,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
```
Comment (ibuziuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: could we have it set to true

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 4]
```diff
@@ -3642,168 +3643,185 @@ data:
...
+            "min": false,
```
Comment (ibuziuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: could we have it set to true

Comment (tomgeorge) [reply deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: Changed

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml (hunk 3 above)
Comment (tomgeorge) [reply deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: Chagned

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml (hunk 2 above)
Comment (tomgeorge) [reply deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: Chagned

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml (hunk 1 above)
Comment (tomgeorge) [reply deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: Changed

**Comment by skabashnyuk** [comment]
May I suggest an alternative approach?
Знімок екрана  о 08 47 28
I would like to propose
 -  leave area ```1``` as an indicator area.  With values that are simple to read.
Like just numbers without graphs.
- Move a total number of workspaces to workspace area - ```2```. Here we can make it as a graph with min, max, avg values
- Make a user panel somewhere below. We can add total number of uses, later we can add online aka active users number.

**Comment by ibuziuk** [comment]
@skabashnyuk sounds like a good idea @tomgeorge wdyt ?

**Comment by tomgeorge** [comment]
I can make that change

**Comment by tomgeorge** [comment]
Pushed the suggested changes

cc @ibuziuk @skabashnyuk

**Comment by skabashnyuk** [comment]
Can you provide a screenshot of the result?

**Comment by skabashnyuk** [comment]
−415  lines removed. Are you sure I have the latest https://github.com/eclipse/che/pull/14578 changes ?

**Comment by tomgeorge** [comment]
No looks like I messed that up.  I need to fix this branch.

**Comment by tomgeorge** [comment]
GH was showing only 2 commits in the PR view compared to https://github.com/eclipse/che/compare/master...tomgeorge:che-14155 which was showing 3 commits.  Looks like the state got out of whack on the GH side.  Changing the base branch to 7.0.x and back to master seems to show the correct diff.  Sorry!

**Comment by tomgeorge** [comment]
General panel
general-panel

Workspaces panel
workspaces-panel

Users panel 
users-panel

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 5]
```diff
@@ -3721,7 +3727,7 @@ data:
               "value": "null"
             }
           ],
-          "valueName": "current"
+          "valueName": "avg"
```
Comment (ibuziuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: @tomgeorge so, currently the `Number of Users` / `Number of Workspaces` will show the average for the given timeframe? Not sure if it makes sense... I would opt for Current number of users & workspaces

Comment (tomgeorge) [reply deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: Good catch, changed back to current.

**Review by ibuziuk — APPROVED** [review]
LGTM
@skabashnyuk could you please provide code-owner review / approval ?

**Review by amisevsk — APPROVED** [review]
LGTM

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 6]
```diff
@@ -4652,6 +4663,97 @@ data:
...
+          "xaxis": {
+            "buckets": null,
+            "mode": "time",
+            "name": null,
+            "show": true,
+            "values": []
+          },
+          "yaxes": [
+            {
+              "format": "short",
```
Comment (skabashnyuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml:4738]: "decimals": 0,
maybe it makes sense to add "decimals=0"

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 7]
```diff
@@ -4660,6 +4762,109 @@ data:
...
```
Comment (skabashnyuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml:4844]: what about "decimals": 0, ?

**Review by skabashnyuk — APPROVED** [review]
Looks good to me.

**Comment by tomgeorge** [comment]
@skabashnyuk I changed the formatting of those panels to have zero precision

**Comment by ibuziuk** [comment]
ci-build

**Pull request merged**